CHANGELOG.md

```diff
@@ -2,6 +2,142 @@
+
+#### 🕸️ Web
+- Load examples manifest via HTTP [#4391](https://github.com/rerun-io/rerun/pull/4391)
+- Remove builds and usage of `demo.rerun.io` [#4418](https://github.com/rerun-io/rerun/pull/4418)
+- Open all links in a new tab [#4582](https://github.com/rerun-io/rerun/pull/4582)
+
+#### 🎨 Renderer Improvements
+- Log wgpu adapter on web [#4414](https://github.com/rerun-io/rerun/pull/4414)
+- Interior mutability for re_renderer's static resource pools (RenderPipeline/Shader/Layouts/etc.) [#4421](https://github.com/rerun-io/rerun/pull/4421)
+- Make draw data creation no longer require a mutable re_renderer context [#4422](https://github.com/rerun-io/rerun/pull/4422)
+- Move re_renderer examples to its own crate in order to make workspace level examples less confusing [#4472](https://github.com/rerun-io/rerun/pull/4472)
+- Improved wgpu error handling, no more crashes through wgpu validation errors [#4509](https://github.com/rerun-io/rerun/pull/4509)
+- Expose `wgpu` profiling scopes to puffin [#4581](https://github.com/rerun-io/rerun/pull/4581)
+- Improve shading with two lights instead of one [#4648](https://github.com/rerun-io/rerun/pull/4648)
+
+#### 🧑‍💻 Dev-experience
+- Fix not tracking wgsl file changes for web build [#4374](https://github.com/rerun-io/rerun/pull/4374)
+- Auto format all the things [#4373](https://github.com/rerun-io/rerun/pull/4373)
+- Refactor naming of `SpaceViewClass` and changed `TextSpaceView` name to "Text Log" [#4386](https://github.com/rerun-io/rerun/pull/4386)
+- Local-first wheel publishing [#4454](https://github.com/rerun-io/rerun/pull/4454)
+
+#### 🗣 Refactors
+- Selection state is now fully double buffered and has interior mutability [#4387](https://github.com/rerun-io/rerun/pull/4387)
+- Time control is now behind a RwLock, making recording config access non-mutable everywhere [#4389](https://github.com/rerun-io/rerun/pull/4389)
+- Enable (selected) new cargo clippy lints [#4404](https://github.com/rerun-io/rerun/pull/4404)
+- Add lint for builder pattern functions and deref impls to be marked `#[inline]` [#4435](https://github.com/rerun-io/rerun/pull/4435)
+- RenderContext usage cleanup [#4446](https://github.com/rerun-io/rerun/pull/4446)
+- Integrate re_tensor_ops crate into re_space_view_tensor [#4450](https://github.com/rerun-io/rerun/pull/4450)
+- Rename `StoreDb` to `EntityDb`, `re_data_store` -> `re_entity_db` [#4670](https://github.com/rerun-io/rerun/pull/4670)
+- Rename `re_arrow_store` to `re_data_store` [#4672](https://github.com/rerun-io/rerun/pull/4672)
+
+#### 📦 Dependencies
+- Update egui and wgpu [#4111](https://github.com/rerun-io/rerun/pull/4111)
+- Update Rust to 1.74.0 [#4390](https://github.com/rerun-io/rerun/pull/4390)
+
+#### 🤷‍♂️ Other
+- Use `:` instead of `.` as the entity:component separator in paths [#4471](https://github.com/rerun-io/rerun/pull/4471)
+- File-like entity paths [#4476](https://github.com/rerun-io/rerun/pull/4476)
```

Comment:
this is so central to our product that I think we should point it out at the top. Not exactly a "highlight" but important to know that entity path handling and escaping changed a bit.